recipe/meta.yaml

```diff
@@ -2,82 +2,138 @@
 {% set version = "0.4.3" %}
```

Comment:
If it's possible to update to **0.4.4** it would be nice, no big changes https://github.com/interpretml/interpret/compare/v0.4.3...v0.4.4. It's up to you